### PR TITLE
Update channelType.md

### PR DESCRIPTION
**Pull request by CorruptedCas**
Added the dm channel type

### CODE REVIEW

File: src/bdscript/channelType.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Returns channel type.
 ```
 $channelType[channel ID]
 ```
-> The different channel types that the bot will retured are: `text`, `voice`, `category`, `thread`, `stage`, `announcement` and `forum`.
+> The different channel types that the bot will retured are: `text`, `voice`, `category`, `thread`, `dm`, `stage`, `announcement` and `forum`.
```
Comment (priyanujgogoi-28) [inline src/bdscript/channelType.md]:
```suggestion
> The different channel types that the bot will return are: `text`, `voice`, `category`, `thread`, `dm`, `stage`, `news` and `forum`.
```

**Review by priyanujgogoi-28 — CHANGES_REQUESTED** [review]

**Review by priyanujgogoi-28 — APPROVED** [review]
Thank you once again!

File: src/bdscript/channelType.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Returns channel type.
 ```
 $channelType[channel ID]
 ```
-> The different channel types that the bot will retured are: `text`, `voice`, `category`, `thread`, `stage`, `announcement` and `forum`.
+> The different channel types that the bot will return are: `text`, `voice`, `category`, `thread`, `dm`, `stage`, `news` and `forum`.
```
Comment (MineBartekSA) [inline src/bdscript/channelType.md]: The News channel type does not exist
```suggestion
> The different channel types that the bot will return are: `text`, `voice`, `category`, `thread`, `dm`, `stage`, `announcement` and `forum`.
```

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

File: src/bdscript/channelType.md (hunk 2 above)
Comment (priyanujgogoi-28) [reply src/bdscript/channelType.md]: Well BDFD return `announcement` as `news` tho

Screenshot_20221101_003825

Comment (MineBartekSA) [reply src/bdscript/channelType.md]: It will change soon

Comment (priyanujgogoi-28) [reply src/bdscript/channelType.md]: Oh! well...Thanks for the info.

**Review by MineBartekSA — APPROVED** [review]
Thank you for your contribution

**Pull request merged**